console/console-init/ui/src/modules/project-detail/utils/utils.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+export const getCertificateLabelByKey = (key: string) => {
```

Comment:
we don't need separate label function , we have aleady function in top lebel utils , getLabelBykey. you can add missing key label in that functuon.

Reply:
Created a separate function as having a common label function at top level might make it overpopulated. Should I use the common one anyway?